Yafc.Model.Tests/Yafc.Model.Tests.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>net8.0</TargetFramework>
-    <PlatformTarget>x64</PlatformTarget>
+    
```

Comment:
Also need to remove `PlatformTarget` from:
  - `Yafc/Yafc.cspro`
  - `CommandLineToolExample/CommandLineToolExample.csproj` (probably, at some point?)

Reply:
What will it mean for other platforms that we remove `PlatformTarget`?

Reply:
Based on [this](https://stackoverflow.com/questions/68825789/relationship-between-platformtarget-and-runtimeidentifier-in-net-5) discussion, `PlatformTarget` just limits on what processors the app can run. In other words, we indeed can remove it.